src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBMetricsProvider.cs

```diff
@@ -93,45 +79,6 @@ public async Task<CosmosDBTriggerMetrics> GetMetricsAsync()
                 partitionCount = 1;
                 remainingWork = 1;
             }
-            catch (Exception e) when (e is CosmosException || e is InvalidOperationException)
-            {
-                if (!TryHandleCosmosException(e))
-                {
-                    _logger.LogWarning(Events.OnScaling, "Unable to handle {0}: {1}", e.GetType().ToString(), e.Message);
-                    if (e is InvalidOperationException)
-                    {
-                        throw;
-                    }
-                }
-            }
-            catch (System.Net.Http.HttpRequestException e)
-            {
-                string errormsg;
-
-                var webException = e.InnerException as WebException;
-                if (webException != null &&
-                    webException.Status == WebExceptionStatus.ProtocolError)
-                {
-                    string statusCode = ((HttpWebResponse)webException.Response).StatusCode.ToString();
-                    string statusDesc = ((HttpWebResponse)webException.Response).StatusDescription;
-                    errormsg = string.Format("CosmosDBTrigger status {0}: {1}.", statusCode, statusDesc);
-                }
-                else if (webException != null &&
-                    webException.Status == WebExceptionStatus.NameResolutionFailure)
-                {
-                    errormsg = string.Format("CosmosDBTrigger Exception message: {0}.", webException.Message);
-                }
-                else
-                {
-                    errormsg = e.ToString();
-                }
-
-                _logger.LogWarning(Events.OnScaling, errormsg);
```

Comment:
I don't understand - why won't these error messages written to the supplied ILogger go to the customer's App Insights or other log streams? The logger should already be configured to do this.